### PR TITLE
Pygments keywords

### PR DESCRIPTION
**Pull request by akrmn**
While working on https://github.com/digital-asset/daml/pull/14743, I noticed a few keywords were not highlighted in the generated html

### CODE REVIEW

File: docs/configs/static/pygments_daml_lexer.py [hunk 1]
```diff
@@ -11,7 +11,30 @@ class DamlLexer(HaskellLexer):
...
-    daml_reserved = ('template', 'with', 'controller', 'can', 'ensure', 'daml', 'observer', 'signatory', 'agreement', 'controller', 'nonconsuming', 'return', 'this')
+    daml_reserved = (
+        'agreement',
+        'can',
+        'choice',
+        'controller',
+        'daml',
+        'ensure',
+        'exception',
+        'for',
```
Comment (akrmn) [inline docs/configs/static/pygments_daml_lexer.py:22]: this one is part of `interface instance I for T`

**Comment by stefanobaghino-da** [comment]
Do you have a small screenshot to share about the change?

**Comment by akrmn** [comment]
> Do you have a small screenshot to share about the change?

Sure,

## before
before

## after
after

**Review by remyhaemmerle-da — APPROVED** [review]
Nice. Thanks

**Comment by stefanobaghino-da** [comment]
> Sure! This shows the new keywords,

🙇🏻‍♂️

**Review by stefanobaghino-da — APPROVED** [review]

**Pull request merged**